src/QuickMarcEditor/QuickMarcEditor.js

```diff
@@ -133,8 +134,8 @@ const QuickMarcEditor = ({
   const { unlinkAuthority } = useAuthorityLinking({ marcType, action });
 
   useEffect(() => {
-    setModifiedSinceLastSubmit(modifiedSinceLastSubmit);
-  }, [modifiedSinceLastSubmit, setModifiedSinceLastSubmit]);
+    setMadeChangesSinceValidation(true);
+  }, [formValues, setMadeChangesSinceValidation]);
```

Comment:
I think the `setMadeChangesSinceValidation(true)` should be triggered only when both conditions are met the validation is made and the user has changed something. It's a little confusing since it also sets `true` on mount.

Reply:
The issue is that it's difficult to know if user has changed some value. `final-form` only provides variables like `modifiedSinceLastSubmit` but they are difficult to work with because we don't want to submit an invalid form
I can rename the variable to `isValidatedCurrentValues`. It would make sense to have `false` on mount and every time `formValues` change, and we can set to `true` after validation